src/request.jl

```diff
@@ -1,7 +1,5 @@
-import CPUSummary
-
 function worker_bits()
-  wts = nextpow2(CPUSummary.sys_threads()) # Typically sys_threads (i.e. Sys.CPU_THREADS) does not change between runs, thus it will precompile well.
+  wts = nextpow2(Threads.nthreads()) # Typically sys_threads (i.e. Sys.CPU_THREADS) does not change between runs, thus it will precompile well.
```

Comment:
```suggestion
  wts = nextpow2(StaticInt{Threads.nthreads()}) # Typically sys_threads (i.e. Sys.CPU_THREADS) does not change between runs, thus it will precompile well.
```